app/views/account/valuations/_valuation.html.erb

```diff
@@ -21,7 +21,7 @@
       <% if valuation.trend.direction.flat? %>
         <%= tag.span t(".no_change"), class: "text-gray-500" %>
       <% else %>
-        <%= tag.span format_money(valuation.trend.value) %>
+        <%= tag.span format_money(Money.new(valuation.trend.value, valuation.value_money.currency)) %>
```

Comment:
This will work for nearly all cases, but I think we can address this a bit further upstream and avoid having to do these additional conversions:

https://github.com/maybe-finance/maybe/blob/a5212f0f5e8fda87c9aed63e6d4d42424e9a7cd9/app/models/account/valuation.rb#L46-L51

If I'm not mistaken, I believe we can update to this and things will work fine:

```rb
def create_trend
  TimeSeries::Trend.new \
    current: self.value_money,
    previous: previous_valuation&.value_money,
    favorable_direction: account.favorable_direction
end
```

Reply:
yes, when I tried to do this I, `current` and `previous` need to be the same type, I think that's what was the immediate issue. Is `previous_valuation&.value_money` maybe scalar?